packages/ethereum-contracts/contracts/superfluid/SuperTokenFactory.sol

```diff
@@ -31,13 +27,19 @@ abstract contract SuperTokenFactoryBase is
 
     ISuperToken internal _superTokenLogic;
 
+    /// @notice A mapping from underlying token addresses to canonical wrapper super token addresses
+    /// @dev Reasoning: (1) provide backwards compatibility for existing listed wrapper super tokens
+    /// @dev (2) prevent address retrieval issues if we ever choose to modify the bytecode of the UUPSProxy contract
+    /// @dev NOTE: address(0) key points to the NativeAssetSuperToken on the network.
+    mapping(address => address) internal _canonicalWrapperSuperTokens;
```

Comment:
Please add a comment nearby here so that each time the code impacts code storage layout, one does not forget to update:

https://github.com/superfluid-finance/protocol-monorepo/blob/697044bbe58a5b2b4fe1992d66913f8e7e51df22/packages/ethereum-contracts/contracts/mocks/SuperTokenFactoryMock.sol#L21

This is a MUST practice for doing upgradable proxy code.